xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VaapiEGL.cpp

```diff
@@ -581,8 +581,8 @@ bool CVaapi2Texture::Map(CVaapiRenderPicture* pic)
 
     CEGLAttributes<8> attribs; // 6 static + 2 modifiers
     attribs.Add({{EGL_LINUX_DRM_FOURCC_EXT, static_cast<EGLint>(layer.drm_format)},
-      {EGL_WIDTH, static_cast<EGLint>(width)},
```

Comment:
this can break on systems where EGLint != int
@pkerling ?

Reply:
width is already of type EGLint
https://github.com/xbmc/xbmc/blob/b5069cec5e44bf554be74700adfd596f9403f451/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VaapiEGL.cpp#L553-L554

Reply:
:+1:

probably oversight or later modification